content/python/concepts/random-module/terms/sample/sample.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '.sample()'
+Description: 'Returns a given number of randomly selected items from a given population.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+  - 'Statistics'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.sample()` method takes a population (list, range, set, or string) and a number as arguments. It returns a list with the specified number of items. Each item is randomly selected without duplication from the population.
+
+## Syntax
+
+```pseudo
+random.sample(<population>,<number>)
+```
+
+Where `<population>` is the sequence the items are selected from, and `<number>` is the number of item(s) to be returned.
```

Comment:
```suggestion
The `population` is a sequence of items can either be a [list](https://www.codecademy.com/resources/docs/python/lists), [string](https://www.codecademy.com/resources/docs/python/strings), or [range](https://www.codecademy.com/resources/docs/python/built-in-functions/range). If [sets](https://www.codecademy.com/resources/docs/python/sets) are used, they should be converted to a list or [tuple](https://www.codecademy.com/resources/docs/python/tuples) before being passed to this method.
```

Looks like [newer versions of `random.sample()` aren't so nice to sets](https://docs.python.org/3/library/random.html#:~:text=Deprecated%20since%20version%203.9%3A%20In%20the%20future%2C%20the%20population%20must%20be%20a%20sequence.%20Instances%20of%20set%20are%20no%20longer%20supported.%20The%20set%20must%20first%20be%20converted%20to%20a%20list%20or%20tuple%2C%20preferably%20in%20a%20deterministic%20order%20so%20that%20the%20sample%20is%20reproducible.). Let's err on the side of caution and use the recommendations described in the suggestion above.